### PR TITLE
Full Update TVDB exceptions.txt

### PR DESCRIPTION
**Pull request by mbxIO**
DO NOT MERGE YET 
Fixing repeats, deleting duplicates and updating overall exceptions in general. 

*Currently on Line 27.

### CODE REVIEW

**Comment by mbxIO** [comment]
@miigotu do you trust my judgement at this point or should i continue to comment on my corrections?

**Comment by mbxIO** [comment]
@miigotu will you take a look at my latest update and take a look at _new_ line 73 I believe ... 

"Richard Hammonds Journey To..."

How will SickRage respond to the "..." at the end of "To"? Do you need the exception without the "..." or is that a repeat and I can delete it too?

**Comment by mbxIO** [comment]
@miigotu Once you take a look at the new 100 line, same thing goes for new Line 109 "A.N.T." Compared to "A.N.T" without the . at the end of T. The same for SR?

Line 119 is marked because it has an exception to pick up shows entitled blood and sand but that is the name of another spartacus show called "Spartacus:Blood and Sand - The Motion Comic"

**Comment by miigotu** [comment]
@mbxplex you should try to make a PR for every 5 or so changes, because each change has to be manually verified before I can merge them
You dont need to comment every change unless it is something out of the ordinary, like pointint to the wrong show.

**Comment by mbxIO** [comment]
@miigotu I know I went crazy ... just review them when you can, ask question whenever you have them and I'd be glad to answer (photographic memory). When you can, I added some notes to the last update I made entitled "Update exceptions.txt". There was a few lines I didn't edit because I was unsure about how SR would behave so I didn't want to touch them until you told me otherwise.

**Comment by miigotu** [comment]
Can you delete all the comments "Name of show" and "Already on tvdb'" ? Hard to follow a thread with mass comments =P

**Comment by mbxIO** [comment]
@miigotu Quick question ... what should I do in a situation like this. I thought to use scene numbering, and fill in my own season x episode number but it didnt work. 

capture1
capture2

SR said no results found for 7x8, so it doesn't seem to be picking up on my custom scene numbering. Any suggestions on how to get SR to pull this EP? IDK why it's labeled a season ahead because TVDB is off and TVRage doesn't even list it.

**Comment by miigotu** [comment]
No idea.
